src/account/Settings.tsx

```diff
@@ -445,17 +472,25 @@ export const Account = ({ navigation, route }: Props) => {
 
           <View style={styles.spacer} />
           {numberVerified && (
-            <SettingsExpandedItem
-              title={t('revokePhoneNumber.title')}
-              details={t('revokePhoneNumber.description')}
-              onPress={handleShowConfirmRevoke}
-              testID="RevokePhoneNumber"
-            />
+            <>
+              <SettingsExpandedItem
+                title={t('revokePhoneNumber.title')}
+                details={t('revokePhoneNumber.description')}
+                onPress={handleShowConfirmRevoke}
+                testID="RevokePhoneNumber"
+              />
+              <SettingsExpandedItem
```

Comment:
unverified users don't need this "reset account" button, since the "delete account" will do the same thing for them (and "delete account" is more compliant with app store guidelines)

Reply:
Ah, but aren't users used to this reset account?
Maybe it's fine if these 2 buttons currently do the same thing if the user is not verified.

Reply:
yeah true, okay i can leave both of them always visible